advanced/Scripts/piholeDebug.sh

```diff
@@ -418,7 +418,37 @@ diagnose_operating_system() {
     get_distro_attributes
   else
     # If it doesn't exist, it's not a system we currently support and link to FAQ
-    log_write "${CROSS} ${COL_LIGHT_RED}${error_msg}${COL_NC} (${FAQ_HARDWARE_REQUIREMENTS})"
+    log_write "${CROSS} ${COL_RED}${error_msg}${COL_NC} (${FAQ_HARDWARE_REQUIREMENTS})"
+  fi
+}
+
+check_selinux() {
+  # SELinux is not supported by the Pi-hole
+  echo_current_diagnostic "SELinux"
+  # Check if a SELinux configuration file exists
+  if [[ -f /etc/selinux/config ]]; then
+    # If a SELinux configuration file was found, check the default SELinux mode.
+    DEFAULT_SELINUX=$(awk -F= '/^SELINUX=/ {print $2}' /etc/selinux/config)
+    case "${DEFAULT_SELINUX,,}" in
+      enforcing)
+        log_write "${CROSS} ${COL_RED}Default SELinux: $DEFAULT_SELINUX${COL_NC}"
+        ;;
+      *)  # 'permissive' and 'disabled'
+        log_write "${TICK} ${COL_GREEN}Default SELinux: $DEFAULT_SELINUX${COL_NC}";
+        ;;
+    esac
+    # Check the current state of SELinux
+    CURRENT_SELINUX=$(getenforce)
+    case "${CURRENT_SELINUX,,}" in
+      enforcing)
+        log_write "${CROSS} ${COL_RED}Current SELinux: $CURRENT_SELINUX${COL_NC}"
+        ;;
+      *)  # 'permissive' and 'disabled'
+        log_write "${TICK} ${COL_GREEN}Current SELinux: $CURRENT_SELINUX${COL_NC}";
+        ;;
+    esac
+  else
+    log_write "${TICK} ${COL_GREEN}SELinux not Supported${COL_NC}";
```

Comment:
This is the default message that everyone will see, and I think it could be worded a little better. Perhaps "SELinux not detected"?